hid.pyx

```diff
@@ -369,5 +415,12 @@ cdef class device:
         :raises IOError:
         """
         if self._c_hid == NULL:
-            raise ValueError('not open')
+            raise ValueError("Device is not open")
         return U(<wchar_t*>hid_error(self._c_hid))
+
+
+cdef class device(Device):
+    """Old alias for the `Device` class, use the new version instead!"""
+
+    def __cinit__(self):
+        warnings.warn("Use the `Device` class instead", DeprecationWarning)
```

Comment:
I imagine¹ this warning could be quite annoying for projects that depend on cython-hidapi.

Please consider that when packaged in Linux distros, these projects need to be compatible with many versions of cython-hidapi, since each distro decides which cython-hidapi version to package.

On a related note, could cython-hidapi provide an internal `version` string or tuple?  This would make it easier for dependent projects to manage compatibility, as well as would allow us to start using the context manager (much) sooner.

_¹ For some reason I have yet to figure out, this warning only affects my test suite, not the CLI most of my users  see._

Reply:
Hm, I see your point.

This package is still behind Pip version control, so a package could still have an exact requirement for e.g. `hidapi=0.10.1` (before this merge). Then compatibility is not an issue.

Second, it is only a warning. Existing projects will continue to work, while being encouraged to update to match the latest versions.

Conventions are quite clear on class names, so I would want to keep this change and enforce it.
If we keep the old class name without a warning users will likely never update their own code.

Reply:
Fun fact, `pip` only started to (correctly) enforce those requirements [last](https://blog.python.org/2020/11/pip-20-3-release-new-resolver.html) [November](https://pyfound.blogspot.com/2019/12/moss-czi-support-pip.html).  It might be too soon to assume that everyone can depend on `pip` doing the right thing.

Regardless, any attempt for a dependent project to enforce a stricter version requirement would simply cause it from being dropped from Linux distributions that, for one reason or the other, do not yet ship the new cython-hidapi.  When this is necessary because of serious compatibility issues, it's obviously beneficial; but for a name change, not so much.

The only reasonable ways for dependent packages to deal with the warning, for next couple of years, are to handle it or suppress it entirely.  And handling it requires conditional behavior, either through the proposed `version` attribute (we cannot even assume `pip` is necessarily installed on users computers), or by `try: dev = hid.Device(); except AttributeError: dev = hid.device()`.

Reply:
Alright, we'll create an alias without errors or warnings then.

Reply:
Thanks! : )